planning/costmap_generator/test/test_points_to_costmap.cpp

```diff
@@ -0,0 +1,213 @@
+#include <costmap_generator/points_to_costmap.hpp>
```

Comment:
I think you need to add a copyright notation here.
https://github.com/autowarefoundation/autoware.universe/runs/6588171538?check_suite_focus=true#step:8:309

Reply:
The check is from here.
https://github.com/autowarefoundation/autoware.universe/blob/c88adb101e974ce66679071439328af7b6f234f5/planning/costmap_generator/package.xml#L34
https://github.com/autowarefoundation/autoware_common/blob/d95da5d4c3f262259c6e40a0bdc1399c0fdf612f/autoware_lint_common/package.xml#L17
https://github.com/ament/ament_lint/tree/master/ament_copyright